munich/README

```diff
@@ -1,7 +1,19 @@
 ###PyLadies Munich
 
-The repo for PyLadies Munich. 
+We are the Munich chapter of an international mentorship group, PyLadies, with a focus on helping more women become active participants and leaders in the Python open-source community. Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.
```

Comment:
Style suggestion: Consider splitting this to multiple lines <= 78 characters

Reply:
Yes, done :)